docs/LOGGING.md

```diff
@@ -14,11 +14,11 @@ The current log pipeline can be vertically split into the following three stages
+All containers will now log directly to stdout, which are then picked up by containerd. Containerd then forwards the logs received to memlogd service that maintains a circular buffer of fixed size (5000 messages limit and 8192 bytes limit per message size) for storing log messages. Memlogd provides a way (via unix domain sockets) to read the logs present in it’s buffer and also stream the incoming logs. Log from Pillar services are also forwarded into the memlogd circular buffers.
 
-The imemlogd plugin looks for json messages in the log and then skips the preceding bytes of log if found. This is done to make it possible for rsyslogd to parse the json message and extract fields from message and also add more fields to the existing message. Imemlogd plugin also puts a default tag name of the incoming container name into each of the log messages. For example logs sent to stdout from pillar container are tagged with "pillar.out" and those logs sent to stderr are tagged with "pillar.err" by default. Log messages that are successfully parsed by rsyslogd in json format will be modified to have some of the existing values changed and some new key-value pairs added. For example rsyslogd adds the current boot image partition to json formatted logs.
+The imemlogd plugin looks for json messages in the log and then skips the preceding bytes of log if found. This is done to make it possible for rsyslogd to parse the json message and extract fields from message and also add more fields to the existing message. Imemlogd plugin also puts a default tag name of the incoming container name into each of the log messages. For example logs sent to stdout from pillar container are tagged with "pillar.out" and those logs sent to stderr are tagged with "pillar.err" by default. Log messages that are successfully parsed by rsyslogd in json format will be modified to have some of the existing values changed and some new key-value pairs added.
 
-Logs coming from pillar container that cannot be json parsed or truncated will have their source field set to pillar.out/pillar.err in EVE log API. It is at times useful to look for pillar.out/pillar.err as source match criteria in EVE log API. The same applies to other containers like lisp, xen-tools etc.
+Logs coming from pillar container that cannot be json parsed or truncated will have their source field set to pillar.out/pillar.err in EVE log API. It is at times useful to look for pillar.out/pillar.err as source match criteria in EVE log API. The same applies to other containers like lisp, xen-tools etc. Newlogd needs to sort out the application log messages and find assigned application UUID and appName when writing into log files for applications.
```

Comment:
"needs to" sounds like a future requirement; shouldn't this use current tense?

Reply:
right. will remove this "needs to" here.